bindata/v4.0.0/controller/deployment.yaml

```diff
@@ -18,27 +18,33 @@ spec:
       name: service-ca
       annotations:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
-        openshift.io/required-scc: restricted-v2
+        openshift.io/required-scc: restricted-v3
```

Comment:
but why do we need to set this ?

Reply:
but why do we want to use that policy ?

Reply:
Actually now that I think about it, it's required for user namespaces. Because it explicitly sets the allowed UID range to the one required by user namespaces. Otherwise the UID range is picked up based on the annotation on the associated namespace and that cannot be used for user namespaces. So this is why: 

https://github.com/openshift/cluster-kube-apiserver-operator/blob/main/bindata/bootkube/scc-manifests/0000_20_kube-apiserver-operator_00_scc-restricted-v3.yaml#L40

Reply:
Yeah, exactly. It implies some defaults, but also enforces the spec is not in conflict with the policy.